cmd/krel/cmd/release_notes.go

```diff
@@ -113,12 +150,116 @@ func runReleaseNotes() (err error) {
 	logrus.Infof("Using start tag %v", start)
 	logrus.Infof("Using end tag %v", tag)
 
-	_, err = releaseNotesFrom(start)
+	if releaseNotesOpts.createDraftPR {
+		err = validateDraftPROptions()
+		if err != nil {
+			return errors.Wrap(err, "validating PR command line options")
+		}
+	}
+
+	result, err := releaseNotesFrom(start)
 	if err != nil {
 		return errors.Wrapf(err, "generating release notes")
 	}
 
-	//TODO: implement PR creation for k-sigs/release-notes and k/sig-release
+	// Create RN draft PR
+	if releaseNotesOpts.createDraftPR {
+		err = createDraftPR(tag, result)
+		if err != nil {
+			return errors.Wrap(err, "Failed to create release notes draft PR")
+		}
+	}
```

Comment:
If `releaseNotesOpts.createDraftPR` is false, because `--create-draft-pr` was not given, should we log out to say we're skipping it and why? Might help users understand why it's not working if they missed that flag.

Reply:
I think if do not specify an output option for the release notes we could print out the help screen. There are and will be more options in addition to --create-draft-pr in the subcommand. For example just writing the release notes to files or generating the json version.